packages/itwin/ec3-widget/src/components/EC3/EC3Config.ts

```diff
@@ -61,6 +76,11 @@ export interface EC3ConfigCommonProps {
    * A custom implementation of EC3ConfigurationClient. If provided, carbonCalculationBasePath is ignored.
    */
   ec3ConfigurationsClient?: IEC3ConfigurationsClient;
+
+  /**
+   * A custom implementation of EC3ConfigurationClient. If provided, carbonCalculationBasePath is ignored.
+   */
+  groupingsClient?: IGroupsClient;
```

Comment:
Why is the groups client needed here? EC3 API operates on Reports, not on groups.

Reply:
Removed all code related to grouping